### ISSUE TITLE
Any V1 forks that support Django 3x?

### ISSUE THREAD

**Issue by yigitguler** [darklow/django-suit#781]
Hi,

Are there any version 1 forks that support Django 3.1 or 3.2? It was a great project, it can be good to list maintained forks here.

**Comment by tjerkw** [comment]
If you follow this thread: https://github.com/darklow/django-suit/issues/475#issuecomment-859598128
And especially this comment:
https://github.com/darklow/django-suit/issues/475#issuecomment-856635914

You can see that @mhindery created a fork of V1 with some fixes for Django 3:

https://github.com/mhindery/django-suit/tree/develop

I didnt try it, but maybe we can all start using it, and add fixes to it?

**Comment by tjerkw** [comment]
@darklow can you hand over this project? Or are you ok with us forking it?

**Comment by yigitguler** [comment]
Thank you @tjerkw. If we see any issues or improvement chances, we will open PRs to @mhindery's fork.

**Comment by pulse-mind** [comment]
For your information, on my fork v2 I made e'w commits
I am working on Bootstrap 5.1 branch too

**Comment by darklow** [comment]
I can easily run Django 3.x with following fixes: 
https://github.com/darklow/django-suit/issues/763#issuecomment-982992027

**Comment by mheijink** [comment]
> For your information, on my fork v2 I made e'w commits I am working on Bootstrap 5.1 branch too

Hi pulse-mind, can you adjust your fork so it also support Django 4 ?

**Comment by pulse-mind** [comment]
> > For your information, on my fork v2 I made e'w commits I am working on Bootstrap 5.1 branch too
> 
> Hi pulse-mind, can you adjust your fork so it also support Django 4 ?

Yes, I will do it, but I always wait a little bit when a new release is coming.

**Comment by mhindery** [comment]
I'll also try to get my fork working with the 4.x changes (since we use it at my company and like to upgrade fast, it should be on the short term timeline)

**Comment by pulse-mind** [comment]
I just pushed a new branch for v4 "django40". Tests on local are ok. I added github workflow...

**Comment by pulse-mind** [comment]
It looks good and it should works also with django v3. The migration is very light to have it working on Django v4

**Comment by pulse-mind** [comment]
@mhindery 
I've done the pull request + merge on my V2 branch. 
It works with Django 3 and Django 4.

**Comment by Elger9** [comment]
> It works with Django 3 and Django 4.

Hi, thanks for the fork for Django Suit for Django 3x and 4x. I am just trying it https://github.com/pulse-mind/django-suit/tree/v2 with Django 3.2.16 LTS - installed nicely. However, I noticed a bug. I don't have the green 'Add modelname' button next to my models in change list view. It looks like there is no 'toolbar' div. The button appears next to models from Django ie auth.user or auth.group. Have you encountered this problem? Greetings.

**Comment by pulse-mind** [comment]
@Elger9 Did you try the demo of this version ? 
I suggest you to run the demo on your PC to check if these problems appear also on the demo ;). Let me know.

**Comment by Elger9** [comment]
Thanks for the fast reply. The solution was simple. You need to define `search_fields` in class admin.ModelAdmin. Then the appropriate layer in HTML is attached, where the add button is.

By the way, it would be nice if the author would add commits from your fork to base repo.

**Comment by pulse-mind** [comment]
The author is working on some other things and does not maintain anymore his repo. I think he does not want to be responsible and spend time on that, I can understand !
Have a good day.

**Comment by Elger9** [comment]
Thanks for the clarification, have a nice day as well.
